provider_counter/lib/main.dart

```diff
@@ -23,6 +28,25 @@ void main() {
   );
 }
 
+const double kWindowWidth = 360;
+const double kWindowHeight = 640;
```

Comment:
Actually I just realised this is an issue: https://dart.dev/guides/language/effective-dart/style#dont-use-prefix-letters

can you rip these out from your other updates as well?

Reply:
yes, will do that